examples/nodejs-simple/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@google-cloud/agones-sdk": "1.3.0"
+    "@google-cloud/agones-sdk": "1.4.0"
```

Comment:
Perhaps should update as part of the release @markmandel ?

Reply:
I don't think this is necessary, unless there is a code change in either the SDK or the example.

Unless you can see a good reason to?

Reply:
The main reason would be to keep the code sample up to date and consistent with the release. We could also change to a non-pinned dependency

Reply:
But if there is no public API code change between versions, then it is up to date with the latest release :smile: